app/views/dvr/cannotRaiseChallenge.scala.html

```diff
@@ -23,7 +23,7 @@
 @(model: CanChallengeResponse, address: String, localAuth: String, homePageUrl: String, authorisationId: Long, backLinkUrl: String)(implicit request: Request[_], messages: Messages, config: ApplicationConfig)
 
 @mainLayout(
-    pageTitle = messages("cannotRaiseChallenge.title"),
+    pageTitle = messages("cannotRaiseChallenge.title.fail"),
```

Comment:
I don't think you should add the `.fail` suffix. it is the cannotRaiseChallenge page, and this is its `title`. Why `.fail`?

Reply:
@PeterPerhac Updated